Sources/swift-parser-cli/Commands/PerformanceTest.swift

```diff
@@ -37,19 +41,39 @@ struct PerformanceTest: ParsableCommand {
       .filter { $0.pathExtension == "swift" }
       .map { try Data(contentsOf: $0) }
 
-    let start = Date()
+    var incrementalParseTransition: IncrementalParseTransition? = nil
+    var incrementalParseAffectRangeCollector: IncrementalParseNodeAffectRangeCollector? = nil
+
+    var totalTime: TimeInterval = .zero
     let startInstructions = getInstructionsExecuted()
-    for _ in 0..<self.iterations {
+    var previousTreeDict: [Data: (SourceFileSyntax, IncrementalParseNodeAffectRangeCollector?)] = [:]
+    for iter in 0..<self.iterations {
       for file in files {
+        if incrementalParse == true {
+          incrementalParseAffectRangeCollector = previousTreeDict[file]?.1 ?? IncrementalParseNodeAffectRangeCollector()
+          if iter != 0 {
+            incrementalParseTransition = IncrementalParseTransition(
+              previousTree: previousTreeDict[file]!.0,
+              edits: ConcurrentEdits(fromSequential: []),
+              reusedNodeDelegate: nil
+            )
+          }
```

Comment:
Add an initial parse is a great idea! But I still store a `(SourceFileSyntax, IncrementalParseNodeAffectRangeCollector)` here since I think we could not get the collector from `IncrementalParseTransition`.